templates/README.md

```diff
@@ -1,44 +1,69 @@
 ﻿# Byndyusoft.Template.Api
 ASP.NET Core MVC tracing.
 
-[![Nuget](http://img.shields.io/nuget/v/Byndyusoft.Template.Api.svg?maxAge=10800)](https://www.nuget.org/packages/Byndyusoft.Template.Api/) [![NuGet downloads](https://img.shields.io/nuget/dt/Byndyusoft.Template.Api.svg)](https://www.nuget.org/packages/Byndyusoft.Template.Api/) 
-
-
-## Installing
-
-```shell
-dotnet add package Byndyusoft.Template.Api
-```
-
 ## Usages
 
 ```shell
 Sample
 ```
 
-# Contributing
-
-To contribute, you will need to setup your local environment, see [prerequisites](#prerequisites). For the contribution and workflow guide, see [package development lifecycle](#package-development-lifecycle).
-
-A detailed overview on how to contribute can be found in the [contributing guide](CONTRIBUTING.md).
-
 ## Prerequisites
 
 Make sure you have installed all of the following prerequisites on your development machine:
 
 - Git - [Download & Install Git](https://git-scm.com/downloads). OSX and Linux machines typically have this already installed.
 - .NET Core (version 6.0 or higher) - [Download & Install .NET Core](https://dotnet.microsoft.com/download/dotnet-core/6.0).
 
+## Configuration
+
+### ConnectionStrings
+Database connection settings.
+
+Example:
+```json
+"ConnectionStrings": {
+    "Main": "Server=localhost;Port=5432;Database=localhost;User Id=user1;Password=password1;POOLING=True;MINPOOLSIZE=1;MAXPOOLSIZE=1024;"
+  }
+```
+
+### Logging
+Logging settings.
+
+Example:
+```json
+"Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+```
+
+### Jaeger
+Jaeger connection settings.
+
+Example:
+```json
+"Jaeger": {
+    "AgentHost": "localhost",
+    "AgentPort": 6831
+  }
+```
+
 ## General folders layout
 
 ### src
-- source code
 
-### tests
-- unit-tests
+- Api - Web API Applications
+- Api.Client - Client for Web API, extension for client registration in consumer
```

Comment:
Web API client and DI registration for consumers?

Reply:
++